showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITUnaryCallable.java

```diff
@@ -100,11 +101,38 @@ public void testGrpc_shutdown() {
   }
 
   @Test
-  public void testHttpJson() {
+  public void testHttpJson_receiveContent() {
     assertThat(echoHttpJson("http-echo?")).isEqualTo("http-echo?");
     assertThat(echoHttpJson("http-echo!")).isEqualTo("http-echo!");
   }
 
+  /*
+  This tests has the server return an error back as the result.
+  We use 404 NOT_FOUND Status as that has the same gRPC <-> HttpJson code mapping
+  (showcase sever has a map that translates the code). The showcase server expects
+  a gRPC Status Code and the result is the HttpJson's mapped value
+   */
+  @Test
+  public void testHttpJson_serverResponseError_throwsException() {
+    StatusCode.Code notFoundStatusCode = StatusCode.Code.NOT_FOUND;
+    ApiException exception =
+        assertThrows(
+            ApiException.class,
+            () ->
+                httpJsonClient.echo(
+                    EchoRequest.newBuilder()
+                        .setError(Status.newBuilder().setCode(notFoundStatusCode.ordinal()).build())
```

Comment:
nit: this may not work but can the inner `build()` call be omitted when we nest builders? i.e. does `EchoRequest.newBuilder().setError(Status.newBuilder().setCode(notFoundStatusCode.ordinal())).build()` work?

Reply:
Moved it out to a variable